content/faculty/honorboard.md

```diff
@@ -0,0 +1,75 @@
+title: Доска почёта
+slug: honorboard
+category: faculty
+tags: [_nav, _category_header, faculty]
+submenu_index: 4
+---
+
+<ul class="nav nav-tabs" id="myTab">
+   <li class="active"><a href="#tab1" data-toggle="tab">Сотрудники</a></li>
+  <li><a href="#tab2" data-toggle="tab">Обучающиеся</a></li>
+</ul>
+ 
+<div class="tab-content">
+  <div class="tab-pane active" id="tab1">
+  
+  
+<table class="honor_table">
```

Comment:
Though I'm going to replace the `table` myself, do yourself a favor: learn some CSS layout and stop using tables. http://ru.learnlayout.com/ would be a really good start.